### PR TITLE
Remove free-2016 flag check & group requests

### PR DESCRIPTION
**Pull request by ransombriggs**

### CODE REVIEW

File: commands/ps/index.js [hunk 1]
```diff
@@ -123,25 +103,26 @@ function printDynos (dynos) {
...
+
+  if (!json && !extended) {
+    promises.app_info = heroku.request({
+      path: `/apps/${context.app}`,
+      headers: {Accept: 'application/vnd.heroku+json; version=3.process-tier'}
     })
+    promises.account_info = heroku.request({path: '/account'})
   }
 
+  let {dynos, app_info, account_info} = yield promises
```
Comment (mnoble) [inline commands/ps/index.js]: Is this going to blow up in the case where `!json && !extended` since `promises` won't have those two keys set?

Comment (ransombriggs) [inline commands/ps/index.js]: @mnoble no, there are tests for this, plus I was unsure of this myself so I tested manually, they will be undefined

Comment (mnoble) [inline commands/ps/index.js]: Cool 👍

**Comment by mnoble** [comment]
👍

**Pull request merged**